### ISSUE TITLE
accesso da altra linea

### ISSUE THREAD

**Issue by Trapulo** [EOLOlab/EOLOapp#1]
il WS non offre la possibilità di interrogare i dati anche accedendo da una linea non eolo, autenticandosi con la login di rete? permetterebbe di controllare lo stato per esempio da telefono...

**Comment by alberanid** [comment]
Ciao,
no al momento non prevedo di introdurre questa funzionalità.

**Comment by fvalle1** [comment]
Concordo! Sarebbe utile poter avere una visione del traffico anche da fuori!
